sdist/nix/fonts-themes-packages.nix

```diff
@@ -0,0 +1,54 @@
+# Fonts and themes packages
+{ pkgs, breeze-plus, matugen, gabarito }:
+
+let
+  # Custom package derivations
+  # based on AUR analogs
+  breeze-plus-pkg = pkgs.stdenv.mkDerivation {
+    pname = "breeze-plus";
+    version = "6.2.5";
+    src = breeze-plus;
+
+    nativeBuildInputs = [ pkgs.makeWrapper ];
+
+    installPhase = ''
+      mkdir -p $out/share/icons
+      cp -r src/breeze-plus* $out/share/icons/
+    '';
+  };
+
+  gabarito-pkg = pkgs.stdenv.mkDerivation {
```

Comment:
While a reasonable approach, you can get this more easily from nixpkgs' `google-fonts`.